spec/requests/api/drops_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+
+RSpec.describe Api::DropsController, :vcr => {:cassette_name => "place" }, type: :request do
```

Comment:
You should change the name of cassette. `place` is already used in the other test so this call will overwrite it.